packages/input-text/src/TextInput.js

```diff
@@ -5,15 +5,37 @@ import disabledTooltip, {
   disabledTooltipProps
 } from "@crave/farmblocks-hoc-disabled-tooltip";
 import formInput, { formInputProps } from "@crave/farmblocks-hoc-input";
+import Link from "@crave/farmblocks-link";
 
 import Container from "./styledComponents/TextInput";
 const EnhancedInput = compose(disabledTooltip, formInput)("input");
 
-const TextInput = props => (
-  <Container>
-    <EnhancedInput {...props} displayBlock />
-  </Container>
-);
+const searchProps = {
+  icon: "wg-search",
+  clearable: true
+};
+
+const TextInput = props => {
+  if (props.type.toLowerCase() === "search")
+    props = { ...searchProps, ...props };
```

Comment:
not cool to mutate a parameter 😢 
We can add a eslint rule to check this situation and warn us (: